toolkit/scripts/pkggen.mk

```diff
@@ -51,12 +50,21 @@ clean-cache:
 	$(SCRIPTS_DIR)/safeunmount.sh "$(cache_working_dir)" && \
 	rm -rf $(cache_working_dir)
 
+# Optionally generate a summary of any blocked packages after a build.
+analyze-built-graph: $(go-graphanalytics) $(RPMS_DIR)
```

Comment:
It it intended that this will cause a full rebuild? If not, you should probably omit $(RPMS_DIR). This is a phony target so it will always re-run regardless of if anything has changed.